phpunit/LDAP/AuthLdapTest.php

```diff
@@ -374,39 +374,39 @@ public function testBuildLdapFilter()
         /** @var \AuthLDAP $ldap */
         $ldap = getItemByTypeName('AuthLDAP', 'LDAP3');
         $result = \AuthLDAP::buildLdapFilter($ldap);
-        $this->assertSame("(& (email=*) )", $result);
+        $this->assertSame("(&(email=*) )", $result);
 
         $_SESSION['ldap_import']['interface'] = \AuthLDAP::SIMPLE_INTERFACE;
         $_SESSION['ldap_import']['criterias'] = ['name'        => 'foo',
             'phone_field' => '+33454968584',
         ];
         $result = \AuthLDAP::buildLdapFilter($ldap);
-        $this->assertSame('(& (LDAP3=*foo*)(phonenumber=*+33454968584*) )', $result);
+        $this->assertSame('(&(LDAP3=*foo*)(phonenumber=*+33454968584*) )', $result);
 
         $_SESSION['ldap_import']['criterias']['name'] = '^foo';
         $result = \AuthLDAP::buildLdapFilter($ldap);
-        $this->assertSame('(& (LDAP3=foo*)(phonenumber=*+33454968584*) )', $result);
+        $this->assertSame('(&(LDAP3=foo*)(phonenumber=*+33454968584*) )', $result);
 
         $_SESSION['ldap_import']['criterias']['name'] = 'foo$';
         $result = \AuthLDAP::buildLdapFilter($ldap);
-        $this->assertSame('(& (LDAP3=*foo)(phonenumber=*+33454968584*) )', $result);
+        $this->assertSame('(&(LDAP3=*foo)(phonenumber=*+33454968584*) )', $result);
 
         $_SESSION['ldap_import']['criterias']['name'] = '^foo$';
         $result = \AuthLDAP::buildLdapFilter($ldap);
-        $this->assertSame('(& (LDAP3=foo)(phonenumber=*+33454968584*) )', $result);
+        $this->assertSame('(&(LDAP3=foo)(phonenumber=*+33454968584*) )', $result);
 
         $_SESSION['ldap_import']['criterias'] = ['name' => '^foo$'];
         $ldap->fields['condition'] = '(objectclass=inetOrgPerson)';
         $result = \AuthLDAP::buildLdapFilter($ldap);
         $ldap->fields['condition'] = '';
-        $this->assertSame('(& (LDAP3=foo) (objectclass=inetOrgPerson))', $result);
+        $this->assertSame('(&(LDAP3=foo) (objectclass=inetOrgPerson))', $result);
```

Comment:
Maybe this space can also be an issue.
```suggestion
        $this->assertSame('(&(LDAP3=foo)(objectclass=inetOrgPerson))', $result);
```

Reply:
I have absolutely no idea; but seems like examples in the RFC does not add extra spaces; so yes, maybe it could be removed also.